integration/spark-common/src/main/scala/org/apache/spark/sql/catalyst/CarbonDDLSqlParser.scala

```diff
@@ -264,8 +307,152 @@ abstract class CarbonDDLSqlParser extends AbstractCarbonSparkSQLParser {
           s"Carbon Implicit column ${col.column} is not allowed in" +
           s" column name while creating table")
       }
+    }
+  }
+
+  /**
+    * The method parses, validates and processes the index_handler property.
+    * @param tableProperties Table properties
+    * @param tableFields  Sequence of table fields
+    * @return <Seq[Field]> Sequence of table fields
+    */
+  private def processIndexProperty(tableProperties: mutable.Map[String, String],
```

Comment:
I think adding validations at this level is cumbersome, better create the tableinfo object first and then add validations at IndexHandler level.

Reply:
Why not do these validation and processing the corresponding command?

Reply:
> I think adding validations at this level is cumbersome, better create the tableinfo object first and then add validations at IndexHandler level.

Not sure these are handled?

Reply:
It is moved to CarbonParserUtils.scala